docs/.vitepress/vitepress/styles/base.scss

```diff
@@ -182,7 +182,7 @@ li > ol {
       td:nth-child(2) {
         font-family: var(--font-family);
         white-space: pre-wrap;
-        min-width: 30%;
+        min-width: 11.62rem;
```

Comment:
May I ask why this number?

Reply:
![image](https://user-images.githubusercontent.com/23303044/134338955-4ba5c224-1570-460c-8d77-b022553e0d74.png)
38.7*0.3